packages/nodes-base/nodes/Google/Docs/GoogleDocs.node.ts

```diff
@@ -0,0 +1,415 @@
+		const items = this.getInputData();
+		const returnData: IDataObject[] = [];
+		const length = items.length;
+		const qs: IDataObject = {};
+		let responseData;
+		const resource = this.getNodeParameter('resource', 0) as string;
+		const operation = this.getNodeParameter('operation', 0) as string;
+		for (let i = 0; i < length; i++) {
+			try {
+				if (resource === 'document') {
+					//https://developers.google.com/docs/api/reference/rest/v1/documents/create
+					if (operation === 'create') {
+						const title = this.getNodeParameter('title', i) as string;
+
+						const body: IDataObject = {
+							title,
+						};
+
+						responseData = await googleApiRequest.call(
+							this,
+							'POST',
+							'/documents',
+							body,
+							qs,
+						);
+
+						console.log({operation,responseData});
+
+					}
+					//https://developers.google.com/docs/api/reference/rest/v1/documents/get
+					if (operation === 'get') {
+						const documentId = this.getNodeParameter('documentId', i) as string;
+
+						responseData = await googleApiRequest.call(
+							this,
+							'GET',
+							`/documents/${documentId}`,
+							{},
+							qs,
+						);
+
+					}
+					//https://developers.google.com/docs/api/reference/rest/v1/documents/batchUpdate
+					if (operation === 'update') {
+
+						const documentId = this.getNodeParameter('documentId', i) as string;
+
+						const updateFields = this.getNodeParameter('updateFields', i) as { writeControl?: IDataObject, requestsUi?: IDataObject };
+
+						const body: IDataObject = {
+							documentId,
+							requests: [],
+						};
+
+						if (updateFields.writeControl) {
+							const writeControl = updateFields.writeControl.writeControlObject as IDataObject;
+							body.writeControl = {
+								[writeControl.control as string]: writeControl.value as string,
+							};
+						}
+
+						if (updateFields.requestsUi) {
+							const {
+								createFooterValues,
+								createHeaderValues,
+								createNamedRangeValues,
+								createParagraphBulletsValues,
+								deleteFooterValues,
+								deleteHeaderValues,
+								deleteNamedRangeValues,
+								deleteParagraphBulletsValues,
+								deletePositionedObjectValues,
+								deleteTableColumnValues,
+								deleteTableRowValues,
+								insertPageBreakValues,
+								insertTableValues,
+								insertTableColumnValues,
+								insertTableRowValues,
+								insertTextValues,
+								replaceAllTextValues,
+							} = updateFields.requestsUi as IDataObject;
+
+							// handle replace all text request
+							if (replaceAllTextValues) {
+								(replaceAllTextValues as IDataObject[]).forEach( replaceAllTextValue => {
+									(body.requests as IDataObject[]).push({
+										replaceAllText: {
+											replaceText: replaceAllTextValue.replaceText,
+											containsText: {
+												text: replaceAllTextValue.text,
+												matchCase: replaceAllTextValue.matchCase,
+											},
```

Comment:
Destructuring `replaceAllTextValue` and using shorthand syntax will slim this down. Same for others.

Also no whitespace before the arg: `forEach(replaceAllTextValue` Same for others.